src/R/Support/Impl/Help/Packages/PackageIndex.cs

```diff
@@ -260,7 +262,7 @@ public sealed class PackageIndex : IPackageIndex {
             try {
                 await _host.StartSessionAsync();
                 return _host.LoadedPackageNames;
-            } catch (OperationCanceledException) { }
+            } catch (RException) { }
```

Comment:
I think `StartSessionAsync` can't throw `RException`, but it can throw `RHostDisconnectedException`